src/devices/bus/coco/coco_t4426.h

```diff
@@ -1,9 +1,9 @@
 // license:BSD-3-Clause
 // copyright-holders:Nathan Woods
-#pragma once
+#ifndef MAME_BUS_COCO_T4426_H__
+#define MAME_BUS_COCO_T4426_H__
```

Comment:
This name still contains a double underscore - that makes it reserved.  You can have a trailing single underscore, or remove the trailing underscores altogether.